src/index.ts

```diff
@@ -119,6 +140,19 @@ function parseLineHeight(value: string) {
 	const parsed = parseFloat(value);
 	if (parsed.toString() === value) {
 		return parsed;
+	} else {
+		const match = /^(\+)?(\.)?/.exec(value) as RegExpExecArray;
```

Comment:
Let's add the following to the `tsconfig.json` compiler options:
```json
		"downlevelIteration": true,
```
And then you can remove this `as RegExpExecArray;` here.